container/go/cmd/puller/puller.go

```diff
@@ -94,7 +106,9 @@ func main() {
 		Features:     strings.Fields(*features),
 	}
 
-	pull(*imgName, *directory, platform)
+	if err := pull(*imgName, *directory, *cachePath, platform); err != nil {
+		log.Fatalln("Image pull was unsuccessful: %v", err)
```

Comment:
%v does nothing when used with log.Fatalln. Either remove the %v or use log.Fatalf